ideascube/mediacenter/templates/mediacenter/index.html

```diff
@@ -34,5 +34,5 @@
     {% include 'search/order_by.html' %}
     {% include 'search/filter_by_kind.html' %}
     {% include 'search/filter_by_lang.html' %}
-    {% tag_cloud url='mediacenter:index' model=view.model tags=available_tags %}
+    {% tag_cloud url='mediacenter:index' model=view.model limit=40 tags=available_tags %}
```

Comment:
Proposal: We bump to 100 for now, we work on #701 (the splitting), and we wait for the designers to help us with a UI rework? The UI rework might entail JS (autocompletion, display of chosen tags in "boxes", ...), or a new page "view all tags", or who knows what. At that point we might revisit the 100 limit, lower it down, remove it, or anything else.